### DISCUSSION TITLE
Map LDAP `uidNumber`/`gidNumber` to ID/Access tokens

### DISCUSSION THREAD

**Discussion by king-of-poppk** [dexidp/dex#3318] [Q&A] [answered]
How would one go about including these attributes in OIDC tokens? These attributes do not seem to be listed at https://dexidp.io/docs/custom-scopes-claims-clients/#custom-claims.

**Comment by nabokihms** [comment, marked as answer]
Currently, dex does not support additional claims.

You can try to hack the system and map a uidNumber to the preferred_username claim using the `preferredUsernameAttr` field.

**Reply by king-of-poppk** [reply]
cc @alex-galperin
